addons/sys_intercom/fnc_infantryPhoneRingingPFH.sqf

```diff
@@ -32,7 +32,18 @@ if (count _crew == 0) then {
 if ((isNull _unitInfantryPhone) && {_isCalling} && {alive _vehicle} && {!_noCrew}) then {
     private _position = AGLToASL (_vehicle modelToWorld _infantryPhonePosition); // ACRE_LISTENER_POS is in ASL coordinates
     TRACE_4("Infantry Phone Calling PFH Check",_vehicle,acre_player,_position,_volume);
-    playSound3D [QPATHTO_R(sounds\Cellphone_Ring.wss), objNull, false, _position, 3.16, 1, 75];
+
+    private _soundFile = QPATHTO_R(sounds\Cellphone_Ring.wss);
+    private _volume = 3.16;
+    private _soundPitch = 1;
+    private _distance = 75;
+    if (count (_vehicle getVariable [QGVAR(infPhoneCustomRinging), []]) > 0) then {
+        _soundFile = (_vehicle getVariable QGVAR(infPhoneCustomRinging)) select 0;
```

Comment:
Better to save the `getVariable` result into an entry above if, and just reading that than doing it every time.